ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
If a user write "plain text" with tinyMCE, the resulting html will be `<p>plain text</p>`.
It has no choice to remove the `<p>` tag.
If fact, even if it not a plain text, almost everything is wrapped into a `<p>` tag. `<img>`, `<iframe>` or `<a>` tags are always inside a `<p>` tag and are never direct child of the `<div>`.
Only `<ul>` (and probably `<ol>`) are not wrapped in a `<p>`.

Before tinyMCE, the summary was always (escaped by django) plain text.
This migration tries to mimic what would have been the resulting html text if the user were using tinyMCE but to write plain text.